arch/Kconfig

```diff
@@ -422,6 +427,14 @@ config ARCH_USE_TEXT_HEAP
 		regions for instruction and data and the memory region used for
 		usual malloc doesn't work for instruction.
 
+config ARCH_TLS
```

Comment:
is it enough to control thread_local only through SCHED_THREAD_LOCAL?

Reply:
Yes, I'll remove this flag